circleci/catapult-publish-lambda

```diff
@@ -51,9 +51,11 @@ done
 
 CIRCLE_CI_INTEGRATIONS_URL=$(dirname $CATAPULT_URL)
 
-# hack to switch from /catapult to /v2/catapult
 # TODO: should probably `ark init` things with an environment variable pointing to base url of circle-ci-integrations
-CATAPULT_URL=$(echo "${CATAPULT_URL}" | sed 's/\/catapult/\/v2\/catapult/')
+# Update Catapult URL to v2 if not already
+if [[ ! $CATAPULT_URL = *"/v2/catapult"* ]]; then 
+  CATAPULT_URL=${CATAPULT_URL/catapult/v2/catapult}
```

Comment:
I'm a little confused why this worked - the URL is `${BASE}/v2/catapult` (https://github.com/Clever/circle-ci-integrations/blob/0028020205ec365e4eec2e36eee5813b1cc6a679/swagger.yml#L50) not `/catapult/v2/catapult`)

For example from the `sed` on the left-hand-side:
```
at 14:07:49 ❯ echo 'base/catapult' | sed 's/\/catapult/\/v2\/catapult/'
base/v2/catapult
```

Reply:
This is a find-replace feature in bash, it's not appending in this case. More about it here: https://stackoverflow.com/a/13210909

Reply:
Ooo the `/` is *inside the brackets*, thanks for explaining.